AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ReconcileAndLocalSaveOperation+Action.swift

```diff
@@ -23,10 +23,12 @@ extension ReconcileAndLocalSaveOperation {
 
         /// Operation has applied the incoming RemoteModel to the local database per the reconciled disposition. This
         /// could result in either a save to the local database, or a delete from the local database.
-        case applied(AppliedModel)
+        case applied(AppliedModel, existsLocally: Bool)
```

Comment:
nit: the variable you use later is called `isCreated` while this label is `existsLocally`, can we name this one thing like `case applied(AppliedModel, isNew: Bool)` and the variable `let isNew` in the Resolver?

Reply:
I think `existsLocally` makes more sense? `existsLocally`, `version` and `deleted` together to determine the `MutationType`, otherwise, `isNew` may cause some confusion? I can use `existsLocally ` across places? What do you think?

Reply:
sure to using existsLocally across both places